analysis/codelists.py

```diff
@@ -1,102 +1,71 @@
-from databuilder.codes import REGISTRY, Codelist, codelist_from_csv
-
-
-def codelist(codes, system):
-    code_class = REGISTRY[system]
-    return Codelist(
-        codes={code_class(code) for code in codes},
-        category_maps={},
-    )
-
-
-def combine_codelists(*codelists):
-    codes = set()
-    for codelist in codelists:
-        codes.update(codelist.codes)
-    return Codelist(codes=codes, category_maps={})
+from databuilder.ehrql import codelist_from_csv
 
 
 # A variety of plausible long covid codelists:
-# https://www.opencodelists.org/codelist/opensafely/nice-managing-the-long-term-effects-of-covid-19/64f1ae69/ 
+# https://www.opencodelists.org/codelist/opensafely/nice-managing-the-long-term-effects-of-covid-19/64f1ae69/
 long_covid_nice_dx = codelist_from_csv(
     "codelists/opensafely-nice-managing-the-long-term-effects-of-covid-19.csv",
-    system="snomedct",
     column="code"
 )
 
 
 # https://www.opencodelists.org/codelist/opensafely/referral-and-signposting-for-long-covid/12d06dc0/
 long_covid_referral_codes = codelist_from_csv(
     "codelists/opensafely-referral-and-signposting-for-long-covid.csv",
-    system="snomedct",
     column="code"
-) 
+)
 
 
 # https://www.opencodelists.org/codelist/opensafely/assessment-instruments-and-outcome-measures-for-long-covid/79c0fa8a/
 long_covid_assessment_codes = codelist_from_csv(
     "codelists/opensafely-assessment-instruments-and-outcome-measures-for-long-covid.csv",
-    system="snomedct",
     column="code"
 )
 
-long_covid_combine = combine_codelists(
-  long_covid_nice_dx,
-  long_covid_referral_codes,
-  long_covid_assessment_codes
+long_covid_combine = (
+    long_covid_nice_dx
+    + long_covid_referral_codes
+    + long_covid_assessment_codes
 )
 
 # https://www.opencodelists.org/codelist/opensafely/covid-identification-in-primary-care-probable-covid-sequelae/2020-07-16/
 covid_primary_care_sequelae = codelist_from_csv(
     "codelists/opensafely-covid-identification-in-primary-care-probable-covid-sequelae.csv",
-    system="ctv3",
```

Comment:
I am getting the following error message when trying to run the code. Is there something I need to do to make this new API work or should I add the system argument back in? 

`     Traceback (most recent call last):
       File "/workspace/analysis/dataset_definition_cases.py", line 5, in <module>
         import codelists
       File "/workspace/analysis/codelists.py", line 6, in <module>
         long_covid_nice_dx = codelist_from_csv(
     TypeError: codelist_from_csv() missing 1 required positional argument: 'system'`